labels/en.js

```diff
@@ -1,201 +1,201 @@
 /* eslint-disable */
 module.exports = {
    /* key : label */
-   "Year" : "Year",
```

Comment:
I think this is one of those `eslint` / `prettier` config issues.  can you verify your IDE is using our settings for this file?

Reply:
I think there `should` be the " " in there.

Reply:
Fixed